src/AlgebraicGeometry/Schemes/ProjectiveAlgebraicSet/Objects/Methods.jl

```diff
@@ -15,7 +15,7 @@ end
 function Base.show(io::IO, X::AbsProjectiveAlgebraicSet{<:Field, <:MPolyQuoRing})
   io = pretty(io)
   if get(io, :supercompact, false)
-    print(io, "Scheme")
+    print(io, "Projective algebraic set")
```

Comment:
In the "detailed" printing code above this is an "Algebraic set".

And in the "else" case below, no prefix seems to be shown at all?